features/topic.feature

```diff
@@ -35,5 +35,22 @@ Feature: Create, edit and delete a topic in the course on the HPI SchulCloud pag
 		And <userRole> chooses course with name '<courseName>'
 		Then <userRole> should see that edited topic with name '<changedTopicName>' is shown on the topic list
 		Examples:
-			| userRole | email                           | password     | courseName | topicName | contentTitle  | description         | changedTopicName | changedContentTitle | changedDescription |
-			| teacher   | lehrer@schul-cloud.org | Schulcloud1! | Mathe          | Division     | Operations    | Math operations | Art                         | Picasso                    | Human of Art         |
+			| userRole | email                  | password     | courseName | topicName | contentTitle | description     | changedTopicName | changedContentTitle | changedDescription |
+			| teacher  | lehrer@schul-cloud.org | Schulcloud1! | Mathe      | Division  | Operations   | Math operations | Art              | Picasso             | Human of Art       |
+
+	@deleteTopic
+	Scenario Outline: User delete a topic
+		Given <userRole> logs in with email '<email>' and password '<password>'
+		And <userRole> accepts data protection
+		When <userRole> goes to courses page
+		And <userRole> chooses course with name '<courseName>'
+		And <userRole> adds a topic with name '<topicName>'
+		And <userRole> adds content title '<contentTitle>' and text '<description>' to the topic content
+		Then <userRole> should see that created topic with name '<topicName>' is shown on the topic list
+		And <userRole> adds a new Topic with name '<secondTopicName>'
```

Comment:
Should use "Then" in this line and "And" in the following - both are things to do - no checks any more